src/lib/operators/sort.hpp

```diff
@@ -14,19 +14,28 @@
 
 namespace opossum {
 
+/**
+ * Defines in which order a certain column should be sorted.
+ */
+struct SortColumnDefinition final {
+  SortColumnDefinition(const ColumnID& column, const OrderByMode order_by_mode = OrderByMode::Ascending)
+      : column(column), order_by_mode(order_by_mode) {}
+
+  const ColumnID column;
+  const OrderByMode order_by_mode;
+};
+
 /**
  * Operator to sort a table by a single column. This implements a stable sort, i.e., rows that share the same value will
```

Comment:
Update this part of the comment as well.

Reply:
Resolved in cadf5d739cfc4a713f547ff17b231d1bef9db20f